Engine/src/Engine/MainMenu.cpp

```diff
@@ -0,0 +1,73 @@
+#include "MainMenu.h"
+
+void MainMenu::show()
+{
+	if (ImGui::BeginMainMenuBar())
+	{
+		//sets the font colour to be white
+		s_style->Colors[ImGuiCol_Text] = white;
+
+		//Do not use else-if statements here otherwise
+		//the menu titles will not render correctly
+		//creates a drop down menu where if x is true (open), display y
+		if (ImGui::BeginMenu("File"))
+		{
+			if (ImGui::MenuItem("New", nullptr, nullptr))
```

Comment:
These are nullptr because we have no functionality for them yet, correct?

Reply:
If you can add a comment explaining the same thing, that would be great!

Reply:
@KevinAlexV The method is properly commented in the library definition.

Reply:
@KevinAlexV added another comment explaining it more